test/simple_test.go

```diff
@@ -1,11 +1,25 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 package simple
 
 import (
 	"testing"
 
-	"github.com/golang/protobuf/proto/proto"
-	"github.com/openconfig/gnoi"
-	gbgp "github.com/openconfig/gnoi/bgp"
+	"github.com/golang/protobuf/proto"
+	gpb "github.com/openconfig/gnoi"
```

Comment:
curious -- why do you need to have a label fo this instead of just using `gnoi` ?

Reply:
This is a style-guide rule - we generally always name things `pb` at the end to make it clear that they were generated by a protobuf. In packages that require only one proto, you might see it aliased to just `pb`.